workflow-templates/staticwebapp-node-built-pipeline.properties.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Static Webapp Node build And Deploy Pipeline",
+  "description": "This pipeline will take a stock Static Webapp repo, Use Node to perform build steps and provides a build chain to push a bundle to Artifactory whenever a new commit is pushed to the main branch",
+  "iconName": "octicon browser",
+  "categories": [
+    "deployment",
+    "Node",
+    "static-webapp",
+    "static-website"
+  ],
+  "filePatterns": [
+    "package.json$"
```

Comment:
Is there a better pattern we can use? Maybe an index.html or something? Basically every JS repo is going to have package.json, but only maybe a small number will use this workflow